types/inquirer/index.d.ts

```diff
@@ -90,7 +91,7 @@ declare namespace inquirer {
          * Default value(s) to use if nothing is entered, or a function that returns the default value(s).
          * If defined as a function, the first parameter will be the current inquirer session answers.
          */
-        default?: any | ((answers: T) => any);
+        default?: any | ((answers: T) => any) | ((answers: T) => Promise<any>);
```

Comment:
Because of the leading `any | ...` this type just means `any` to the compiler, and will only display as `any` in quick info etc. It's only useful when users actually read `@types/inquirer/index.d.ts`, so it's up to you if you want to keep it this way or just change it to `any`.

Reply:
Thanks for pointing this out. If it's ok with you (which it sounds like it is), I'd like to keep it as-is because it's not uncommon for people to look at the .d.ts file and it will document the various options available for the default value.